roles/pulp_webserver/tasks/generate_tls_certificates.yml

```diff
@@ -88,3 +88,36 @@
     - pulp_webserver.csr
   loop_control:
     loop_var: __pulp_webserver_csr_path
+
+- name: Create a directory if it does not exist
+  file:
+    path: "{{ pulp_webserver_trusted_root_certificates_path }}/extra"
+    state: directory
+    owner: root
+    group: root
+    mode: '0755'
+  when: ansible_os_family == 'Debian'
+
+- name: Add certificates to trusted
+  copy:
+    src: "{{ pulp_certs_dir }}/root.crt"
+    dest: "{{ pulp_webserver_trusted_root_certificates_path }}/extra/root.crt"
+    owner: root
+    group: root
+    mode: 0644
+    remote_src: yes
+  when: ansible_os_family == 'Debian'
+  become: true
+  notify: update ca trust
+
+- name: Add certificates to trusted
+  copy:
+    src: "{{ pulp_certs_dir }}/root.crt"
+    dest: "{{ pulp_webserver_trusted_root_certificates_path }}/root.crt"
+    owner: root
+    group: root
+    mode: 0644
+    remote_src: yes
+  when: ansible_os_family != 'Debian'
+  become: true
+  notify: update ca trust
```

Comment:
Wouldn't symlinking be better?

Reply:
it would, but no need in this PR anymore